com.yubico.yubioath.json

```diff
@@ -0,0 +1,259 @@
+{
+  "app-id": "com.yubico.yubioath",
+  "runtime": "org.kde.Platform",
+  "sdk": "org.kde.Sdk",
+  "runtime-version": "5.14",
+  "command": "yubioath-desktop",
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig"
+  ],
+  "finish-args": [
+    "--socket=x11",
```

Comment:
this needs --share=ipc to go along with the X11 socket.

Reply:
@nedrichards like this? 2f65c56